recipes/r-metadig/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set name = "r-metadig" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NCEAS/metadig-r/archive/refs/tags/v0.2.1.tar.gz
+  sha256: 031afbd61cd120b9b6428858e0a4d8a2c4bf0b89d2e1abb783bba8b6479ad585
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```

I believe this can be noarch.

```suggestion
  noarch: generic
  number: 0
```